hack/jenkins/common.sh

```diff
@@ -289,6 +301,35 @@ elapsed=$min.$sec
 description="completed with ${status} in ${elapsed} minute(s)."
 echo $description
 
+echo ">> Copying ${TEST_OUT} to gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}out.txt"
+gsutil -qm cp "${TEST_OUT}" "gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}out.txt"
+
+
+echo ">> Attmpting to convert test logs to json"
+rm ${JSON_OUT} || true # clean up build-reruns
+touch ${JSON_OUT}
+PATH=$PATH:"/usr/local/go/bin/" # for gopath
+# Generate JSON output
+go tool test2json -t < ${TEST_OUT} > ${JSON_OUT}
+RESULT=$?
+if [ $RESULT -eq 0 ]; then
+        # Generate HTML human readable test output
+      echo ">> Running gopogh"
+      rm ${HTML_OUT} || true # clean up build-reruns
+      touch ${HTML_OUT}
+      gopogh -in ${JSON_OUT} -out ${HTML_OUT} -name ${JOB_NAME} -pr ${MINIKUBE_LOCATION} -repo github.com/kubernetes/minikube/  -details ${COMMIT} || true
+      echo ">> Copying ${HTML_OUT} to gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}.html"
+      gsutil -qm cp "${JSON_OUT}" "gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}.json"
+      gsutil -qm cp "${HTML_OUT}" "gs://minikube-builds/logs/${MINIKUBE_LOCATION}/${JOB_NAME}.html"
+else
+  echo "go tool test2json is not installed, will skip generating HTML report output."
```

Comment:
A non-zero exit code does not necessarily mean that it was not installed. I would just mention that it failed and that you are skipping the report.